CHANGELOG.md

```diff
@@ -5,6 +5,174 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.12.0] - 2023-07-10
+[v0.12.0]: https://github.com/cilium/hubble/compare/v0.11.6...v0.12.0
+
+**Major Changes:**
+* Add hubble list namespaces command (#1086, @chancez)
+* Add support for supplying basic authentication credentials (#1002, @chancez)
+* Replace stdin detection with --flows-file flag which supports reading flows from a file or stdin (#951, @chancez)
+
+**Minor Changes:**
+* Add experimental-field-mask flags (#1101, @AwesomePatrol)
+* Add hubble observe flows (#851, @chancez)
```

Comment:
This one can be removed, superseded by #875 (mentioned a couple of lines below this one).